### DISCUSSION TITLE
i'm really new to ai I kind of have no idea what i'm doing but if someone could point me in the right direction to what i'm trying to achieve i'd appreciate it

### DISCUSSION THREAD

**Discussion by wawaawawa** [Acly/krita-ai-diffusion#708] [General]
okay so i've been looking to use ai for image editing and enhancing, and i just started using krita and i really love the diverse features it has, but i'm looking to go further in my edits with small image generation to enhance the composition. so for example with this image, what tools should i be using if i wanted to enhance the texture of the background like, creating more bubbles, adding other elements like stars and enhancing the galaxy type background, etc. while still maintaining the lighting and general atmosphere of the image if that makes sense.
I take a lot of in game photos and basically want to use ai to enhance the environment to make more realistic and interesting background textures and elements.

edit: unrelated question but does anyone know of any smart object detection for krita, there's a plugin for windows but not mac >.<

<img width="1273" alt="Screenshot 2024-05-12 at 7 32 59 PM" src="https://github.com/Acly/krita-ai-diffusion/assets/141802568/2d8cff8e-3a70-48bb-9458-5227f35052e0">

**Comment by minsky91** [comment]
A lot depends on how much of change in the image you have generated or picked from a game you are willing to allow, when modifying it with the Krita AI plugin, or any other SD tool for that matter. All kinds of variations of the source image are possible with the Refine feature of the plugin (known also as img2img mode in the SD world), when the refining strength is above 30-35. Experiment with the prompt, various checkpoints, LoRas, styles (you need Cinematic/Photo for realistic renderings), and you will get stars or bubbles or anything else that might make sense as a feature in the background. The problem here is that modifying the image with Refine / img2img will also likely alter the look of the image in a drastic fashion. This is where the Krita's AI plugin makes a lot of difference, compared to more traditional tools. Use selections and layers to restrict the scope of the modification,  and generally try to exercise more control than just driving a slider and then clicking on Generate.      

Most of all, learn from others. Here are some pointers:

https://www.reddit.com/r/StableDiffusion/
https://civitai.com/images
https://www.facebook.com/groups/leonardoaigroup

Not everything is great among the stuff posted in those groups, but there are always gems to be found and prompts & parameters to copy and adapt for your needs.
